vega_sim/api/market.py

```diff
@@ -361,20 +390,40 @@ class InstrumentConfiguration(Config):
             "name": None,
             "code": None,
             "future": "default",
-        }
+            "perp": None,
+        },
+        "perp": {
+            "name": None,
+            "code": None,
+            "future": None,
+            "perp": "default",
+        },
     }
 
     def load(self, opt: Optional[str] = None):
         config = super().load(opt=opt)
 
         self.name = config["name"]
         self.code = config["code"]
-        self.future = FutureProduct(opt=config["future"])
+        self.future = None
+        future_config = config["future"]
+        if future_config != None:
+            self.future = FutureProduct(opt=future_config)
+        self.perp = None
+        perp_config = config["perp"]
+        if perp_config != None:
```

Comment:
Could compress these into `if config["perp"] is not None` (or even a walrus operator if you feel like it)